roles/dns_server/templates/named.conf

```diff
@@ -0,0 +1,66 @@
+// named.conf
+//
+// Provided by Red Hat bind package to configure the ISC BIND named(8) DNS
+// server as a caching only nameserver (as a localhost DNS resolver only).
+//
+// See /usr/share/doc/bind*/sample/ for example named configuration files.
+//
+
+options {
+	listen-on port 53 { any; };
+	directory 	"{{ named_dir }}";
+	dump-file 	"{{ named_dir }}/data/cache_dump.db";
+        statistics-file "{{ named_dir }}/data/named_stats.txt";
+        memstatistics-file "{{ named_dir }}/data/named_mem_stats.txt";
+	allow-query     { any; };
+
+	/* Path to ISC DLV key */
+	bindkeys-file "/etc/named.iscdlv.key";
+
+	recursion no;
+
+};
+
+// use the default rndc key
+include "/etc/rndc.key";
+
+controls {
+	inet 127.0.0.1 port 953
+	allow { 127.0.0.1; } keys { "rndc-key"; };
+};
+
+acl "openstack" { {{ subnet }}; };
+
+view "internal" {
+  match-clients { "openstack"; };
+  recursion yes;
+  include "/etc/named.rfc1912.zones";
+  forwarders {
+{% for server in forwarders %}
+     {{ server }};
+{% endfor %}
```

Comment:
I'd rather see us avoid forwarding and just serve records for the domain(s) we are defining, we can rely on the existing dns servers on the client hosts to handle resolving other domains.